KsApi/mutations/CreatePaymentsSourceMutation.swift

```diff
@@ -12,6 +12,8 @@ public struct CreatePaymentSourceMutation<T: GraphMutationInput>: GraphMutation
     mutation createPaymentSource($input: CreatePaymentSourceInput!) {
     createPaymentSource(input: $input) {
       clientMutationId\
```

Comment:
Is the `\` here correct? What does that mean in GraphQL? I would think that there's no `\` and the three properties should align.

Reply:
Thanks! we do not need this here. I've seen us do this in other schemas. It also seems like GraphQL does not require `\` in schemas so I went ahead and took those out.

Reply:
This was just to add a line break in a multi-line string literal. Not technically needed but it reads nicer when we print the mutations in the debug logs.

Reply:
Thanks Isabel! I think we should keep them since it helps w/ readability on the debug log.